mne/channels/channels.py

```diff
@@ -1340,8 +1340,14 @@ def read_ch_adjacency(fname, picks=None):
             You can retrieve the names of all
             built-in channel adjacencies via
             :func:`mne.channels.get_builtin_ch_adjacencies`.
-    %(picks_all)s
-        Picks must match the template.
+    picks : list
+        Channels to include. Slices and lists of integers will be interpreted
+        as channel indices. In lists, channel name strings (e.g.,
+        ['MEG0111', 'MEG2623'] will pick the given channels.
+        None (default) will pick all channels. Note that channels in
+        info['bads'] will be included if their names or
+        indices are explicitly provided. Picks must match the template.
```

Comment:
would be nice if we could keep this as a docdict-based entry. LMK if you want help with that... it can be a bit complicated to create new entries esp. for picks-related ones.